satellite/metainfo/metainfo.go

```diff
@@ -75,9 +69,8 @@ func NewEndpoint(log *zap.Logger, metainfo *Service, orders *orders.Service, cac
 		cache:                   cache,
 		apiKeys:                 apiKeys,
 		storagenodeAccountingDB: sdb,
-		projectAccountingDB:     pdb,
+		projectUsage:            projectUsage,
```

Comment:
Does Endpoint still need to have the liveAccounting member? `projectUsage` now owns the liveAccounting service, it looks like.

Reply:
good point, `liveAccounting` removed from metainfo